const.go

```diff
@@ -138,6 +142,10 @@ func (a *AccessorType) UnmarshalJSON(data []byte) error {
 			"MAT3":   AccessorMat3,
 			"MAT4":   AccessorMat4,
 		}[tmp]
+		if !ok {
+			return errors.New(fmt.Sprintf("gltf: unknown accessor's type: %s", tmp))
```

Comment:
I missed this yesterday: use `fmt.Errorf` instead of `errors.New` + `fmt.Sprintf`

Reply:
Oh, yes, I've forgotten about this too )) Thank you! Changed